src/pymmcore_widgets/mda/_core_mda.py

```diff
@@ -183,8 +183,8 @@ def get_next_available_path(self, requested_path: Path) -> Path:
         """
         return get_next_available_path(requested_path=requested_path)
 
-    def run_mda(self) -> None:
-        """Run the MDA sequence experiment."""
+    def prepare_mda(self) -> bool | str | None:
+        """Prepare the MDA sequence experiment."""
```

Comment:
looks good to me.  can we just update this docstring to explain how one should interpret the return type?